Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -133,5 +163,18 @@ protected virtual void OnScrollToRequested(ScrollToRequestEventArgs e)
 		{
 			ScrollToRequested?.Invoke(this, e);
 		}
+
+		protected virtual void OnRemainingItemsThresholdReached()
+		{
+			RemainingItemsThresholdReached?.Invoke(this, EventArgs.Empty);
```

Comment:
don't fire events from a virtual method, but fire them from SendRemainingItemsThresholdReached(). same for triggering the Command.

then, it's fine to keep the virtual method with an empty body